source/encoders/encoder-ffmpeg.cpp

```diff
@@ -835,7 +835,7 @@ void ffmpeg_instance::parse_ffmpeg_commandline(std::string text)
 	//  and things like -foo="bar" is stored as std::string("-foo=\"bar\"").
 	// However "-foo"=bar" -foo2=bar" is stored as std::string("-foo=bar -foo2=bar")
 	//  because the quote was not escaped.
-	std::list<std::string> opts;
+	std::vector<std::string> opts;
```

Comment:
`std::vector` is slower than `std::list` if the ideal size of the vector is not known ahead of time.